ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -125,12 +125,12 @@ tests:
 - as: rosa-sts-ad-critical-high-f3
   cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-terraform-qe
     env:
       CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
       CHANNEL_GROUP: candidate
       CLUSTER_PROFILE: rosa-sts-ad
-      REGION: us-east-1
+      REGION: ap-northeast-1
```

Comment:
Please make sure this region can work well.

Reply:
it is working with @AsherShoshan previous job

Reply:
@shellymiron That's the part I am confused with. Can you please run on local with the same AWS credential and same region for the rosa-sts-ad cluster provision? Better to run multiple times, it cannot happen every time.

Reply:
on it :)

Reply:
ok so I ran ~8 different clusters with the `terraform-provider-minimal` creds & this specific region,
seems like it is working as accepted and deployed successfully